pkg/proxy/apis/config/types.go

```diff
@@ -183,15 +183,16 @@ type KubeProxyConfiguration struct {
 	// primary IP is. Note that the name is a historical artifact, and kube-proxy does
 	// not actually bind any sockets to this IP.
 	BindAddress string
-	// healthzBindAddress is the IP address and port for the health check server to
-	// serve on, defaulting to "0.0.0.0:10256" (if bindAddress is unset or IPv4), or
-	// "[::]:10256" (if bindAddress is IPv6).
-	HealthzBindAddress string
-	// metricsBindAddress is the IP address and port for the metrics server to serve
-	// on, defaulting to "127.0.0.1:10249" (if bindAddress is unset or IPv4), or
-	// "[::1]:10249" (if bindAddress is IPv6). (Set to "0.0.0.0:10249" / "[::]:10249"
-	// to bind on all interfaces.)
-	MetricsBindAddress string
+	// healthzBindAddresses is a list of CIDR ranges that contains a valid node IP on which
+	// the healthz server will be served on, defaulting to [ "0.0.0.0/0", "::/0" ].
+	HealthzBindAddresses []string
+	// healthzBindPort is the port on which healthz server will be exposed, defaulting to 10256.
+	HealthzBindPort int32
```

Comment:
this is confusing, so we move from a binding address on the host port format to an array of ips plus a node?

Reply:
We are moving from host:port format to list of cidrs and port format. 
Server will bind on all the nodeIPs which belong to the given list of cidrs.